web/src/components/common/UI/Table.tsx

```diff
@@ -287,6 +287,32 @@ const Table: React.FC<ITableProps> = (props) => {
     [],
   );
 
+  const findRowByRoleInfo = (rolesToSearch: object[], searchKey: string) => {
+    return rolesToSearch.find(
+      (role: any) =>
```

Comment:
Something along the lines of the following:
```interface projectRole {
  firstName: string;
  lastName: string;
  email: string;
}```

The above is off the top of my head so will likely not work exactly as written. 

```suggestion
      (role: [DEFINE THIS]) =>
```

Reply:
I have tried, but always get an error msg saying Types of parameters ''role" and 'value' are incompatible.